recipes/emscripten/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.0.8" %}
+
+package:
+  name: emscripten
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/emscripten-core/emscripten/archive/{{ version }}.tar.gz
+  sha256: 609a7ecea954821efbcc55bf5ad88dd03ba4f836c703b8ad2e37c243329174a6
+  patches:
+    - remove_revision_file_setting.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - binaryen
+    - nodejs
+  run:
+    - python
+    - binaryen
+    - llvmdev
+    - nodejs
```

Comment:
Why do you need `nodejs` here?

Reply:
there are node/npm dependencies and they are invoked in the emscripten build process to minify js as far as i understand